src/context/FeatureContextProvider.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import { Feature, FeatureId } from 'generated/sdk';
+import { useFeatures } from 'hooks/admin/useFeatures';
+
+interface FeatureContextData {
+  readonly features: Map<FeatureId, Feature>;
+}
+
+const initialFeatureData: FeatureContextData = {
+  features: new Map<FeatureId, Feature>(),
+};
+
+export const FeatureContext = React.createContext<FeatureContextData>(
+  initialFeatureData
+);
+
+export const FeatureContextProvider: React.FC = props => {
+  const { features, loadingFeatures } = useFeatures();
+
+  if (loadingFeatures) {
+    return null;
```

Comment:
Maybe using a UOLoader would be more user friendly?

Reply:
Done